videoframes.py

```diff
@@ -106,12 +106,11 @@ def context_menu(self, event):
 
     def toggle_playback(self):
         """Toggles playback of a stream."""
+        self.toggle_button()
```

Comment:
Can we do this after the conditionals? Just a nitpick 😅 Doesn't really matter though, up to you.